modelscope/models/cv/image_matching_fast/lightglue_model.py

```diff
@@ -0,0 +1,80 @@
+# The implementation is made publicly available under the
+# Apache 2.0 license at https://github.com/cvg/LightGlue
+
+import os.path as osp
+from pathlib import Path
+
+import cv2
+import numpy as np
+import torch
+
+from modelscope.metainfo import Models
+from modelscope.models.base.base_torch_model import TorchModel
+from modelscope.models.builder import MODELS
+from modelscope.outputs import OutputKeys
+from modelscope.utils.constant import ModelFile, Tasks
+from .lightglue import LightGlue, SuperPoint, DISK, ALIKED, SIFT
+from .lightglue.utils import rbd, numpy_image_to_torch
+from .config.default import lightglue_default_conf
+
+
+@MODELS.register_module(
+    Tasks.image_matching, module_name=Models.lightglue_image_matching)
+class LightGlueImageMatching(TorchModel):
+
```

Comment:
add docstr for model and paper link if it has

Reply:
Done.